src/afa/tests-afa.cc

```diff
@@ -428,6 +428,139 @@ TEST_CASE("Mata::Afa antichain emptiness test")
 
 }
 
+TEST_CASE("Mata::Afa::construct() from IntermediateAut correct calls")
+{ // {{{
+    Afa aut;
+    Mata::IntermediateAut inter_aut;
+    StringToSymbolMap symbol_map;
+
+    SECTION("construct an empty automaton")
+    {
+        inter_aut.automaton_type = Mata::IntermediateAut::AFA;
+        //REQUIRE(is_lang_empty(aut));
+        aut = Mata::Afa::construct(inter_aut);
+        REQUIRE(true);
+        //REQUIRE(is_lang_empty(aut));
```

Comment:
Could you remove the commented out checks? Otherwise, they will be there forever as no one dares to remove them in case they had some unknown meaning.

Reply:
done